### DISCUSSION TITLE
Plain Black Pop Up

### DISCUSSION THREAD

**Discussion by dev-techguy** [livewire/livewire#5097] [1. Help]
![image](https://user-images.githubusercontent.com/36882600/178157767-45867c95-3062-4f2b-8a0b-72034995a818.png)

**Comment by Martleby** [comment]
How is anyone supposed to help you with this?

What versions are you running?
What code is running?
In what circumstances does this occur?
What is the expected behaviour?
Can you share any code?

**Reply by dev-techguy** [reply]
Hello @Martleby 

I have attached the component and blade

**Reply by dev-techguy** [reply]
`<?php

namespace App\Http\Livewire\Admin;

use App\Models\Analysis;
use LaravelMultipleGuards\Traits\FindGuard;
use Livewire\Component;

class AdminHome extends Component
{
    use FindGuard;

    public bool $readyToLoad = false;

    public function loadData()
    {
        $this->readyToLoad = true;
    }

    public function render()
    {
        return view('livewire.admin.admin-home', [
            'data' => $this->readyToLoad ? Analysis::query()->first() : null,
        ]);
    }
}
`

**Reply by dev-techguy** [reply]
[test.txt](https://github.com/livewire/livewire/files/9138383/test.txt)

**Reply by Martleby** [reply]
From the looks of things, this is a Livewire error popup. I don't know why it's black - do you have some sort of bootstrap overlay or something else that could affect modals? If you inspect the code in the browser's devtools can you see the modal and does it have anything in it?

Why you're getting the error in the first place, I'm unsure. However if 'Analysis' is a model and you're calling first on it, shouldn't you access the model properties like this:

```$data->clients```

rather than:

```$data->data['clients']```

Hope that gives you something to try!

Cheers,
Martin.
